lib/typing/ext/parse_test.go

```diff
@@ -18,7 +18,7 @@ func TestParseFromInterface(t *testing.T) {
 		for _, val := range vals {
 			extTime, err := ParseFromInterface(val, TimestampTZKindType)
 			assert.NoError(t, err)
-			assert.Equal(t, val, extTime)
+			assert.Equal(t, val.(*ExtendedTime).GetTime(), extTime)
```

Comment:
On line 13, can we declare `vals` as `[]ExtendedTime` instead of `[]any` so we don't have to cast `val` here?

Reply:
Oh or will that make `ParseFromInterface` not accept it?

Reply:
It can accept `ExtendedTime`!